spec/server_spec.cr

```diff
@@ -954,4 +954,14 @@ describe LavinMQ::Server do
       s.vhosts["/"].delete_queue(n)
     end
   end
+
+  it "should measure time it takes to collect metrics in stats_loop" do
+    stats_interval = LavinMQ::Config.instance.stats_interval
+    LavinMQ::Config.instance.stats_interval = 100
+    server = LavinMQ::Server.new("/tmp/spec")
+    should_eventually(be_true, 1.seconds) { server.stats_loop_duration_seconds_total > Time::Span.zero }
+    server.stats_loop_rates_duration_seconds.should_not eq Time::Span.zero
+    server.stats_loop_system_duration_seconds.should_not eq Time::Span.zero
+    LavinMQ::Config.instance.stats_interval = stats_interval
```

Comment:
Just a minor detail, if the spec crashes the `stats_interval` won't be restored. So we can put this in a `ensure` block.

```suggestion
  ensure
    LavinMQ::Config.instance.stats_interval = stats_interval if stats_interval
```